src/routes/LPOptimisers/EcosystemDeposit/DepositForm/MaturityDistribution/MaturityDistributionToggle/MaturityDistributionToggle.tsx

```diff
@@ -0,0 +1,61 @@
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import React from 'react';
+
+import { IconLabel } from '../../../../../../components/composite/IconLabel/IconLabel';
+import { doNothing } from '../../../../../../utilities/doNothing';
+import {
+  MaturityDistributionBox,
+  MaturityDistributionInputLabel,
+  MaturityDistributionTypography,
+  ToggleButton,
+} from './MaturityDistributionToggle.styled';
+
+type Distribution = 'automatic' | 'manual';
+
+const distributions: {
+  label: string;
+  id: Distribution;
+}[] = [
+  {
+    id: 'automatic',
+    label: 'AUTOMATIC',
+  },
+  {
+    id: 'manual',
+    label: 'MANUAL',
+  },
+];
+
+export type MaturityDistributionToggleProps = {
```

Comment:
Should we disable the manual toggle if there is only one maturity? It serves no purpose.

Reply:
@0xZenus can you have a look here? This suggestion makes sense for me as well. Going into manual just shows 100% without no ability to edit. Basically if weights.lenght is one we disable input.

Reply:
Done. https://github.com/Voltz-Protocol/voltz-ui/pull/425/commits/b7f6214379be4075fa67f19019c5026bd449ca4d